recipes-bsp/atf/atf_git.bb

```diff
@@ -17,14 +17,22 @@ SRCREV = "4a82c939a0211196e2b80a495f966383803753bb"
 SRC_URI += "file://0001-fix-fiptool-build-error.patch \
     file://0001-Makefile-add-CC-gcc.patch \
 "
+
 COMPATIBLE_MACHINE = "(qoriq)"
+
 PACKAGE_ARCH = "${MACHINE_ARCH}"
+
 PLATFORM = "${MACHINE}"
 PLATFORM_ls1088ardb-pb = "ls1088ardb"
+PLATFORM_ADDITIONAL_TARGET ??= ""
+PLATFORM_ADDITIONAL_TARGET_ls1012afrwy = "ls1012afrwy_512mb"
+
```

Comment:
This improvement should described a bit clearer.
See http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
update commit message.